src/legacy/core_plugins/kibana/public/discover/angular/doc_table/components/table_row.ts

```diff
@@ -59,18 +56,18 @@ module.directive('kbnTableRow', function ($compile, $httpParamSerializer, kbnUrl
       onAddColumn: '=?',
       onRemoveColumn: '=?',
     },
-    link: function ($scope, $el) {
+    link: ($scope: any, $el: any) => {
```

Comment:
`ng.IScope` and `JQuery` should be the correct types here.

Reply:
been there, done that, thx!